added cisco to ethernet section

liljenstolpe opened the pull request:
Added Cisco to the Ethernet section.

liljenstolpe committed:
added cisco to ethernet section

AndreyKozlov1984 commented on landscape.yml:
```diff
@@ -48,6 +48,13 @@ landscape:
+            name: Cisco
+            homepage_url: https://cisco.com/
+            logo: cisco.svg
+            crunchbase: https://crunchbase.com/organization/cisco
```
https://www.crunchbase.com/organization/cisco

liljenstolpe replied:
Ok, it seems as if hte checks are a bit too pedantic, crunchbase works with or without the www.  I can, however, just add the www to the URL.  We should also add cisco to optics.  What is the best way to do this?

liljenstolpe replied:
I have added www to the crunchbase URL and added a second_path for cisco for the optics section.

liljenstolpe committed:
added www to the crunchbase url in cisco to satisfy landscape sanity check

It seems as if the landscape builder is a bit too pedantic about the crunchbase url, and doesn't accept the valid https://crunchbase.com variant.

liljenstolpe committed:
Added Cisco to Optics as well

Added Cisco to the Optics category as well.
